src/routes/api/profile.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import ProfileController from '../../controllers/profileController';
+import middleware from '../../middlewares';
+
+const { Authenticate, validate } = middleware;
+
+const router = Router();
+
+router.route('/')
+  .post(Authenticate.verifyToken, ProfileController.upsertProfile)
```

Comment:
Does it mean that anyone can update any profile even if it is not theirs?

Reply:
So to create we don't need to check if the user is creating his own profile or another person's?

Reply:
In the case of creating, the user isn't sending their `userId`. We are decoding their `userId` from their token. That way, whoever is logged in, is the one whose profile will be created.